dali/test/operators/passthrough_input.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/test/operators/passthrough_input.h"
+
+namespace dali {
+
+DALI_SCHEMA(PassthroughInput)
```

Comment:
Can you hide this from the documentation? It will confuse users. Also, consider putting it in a testing namespace or something.

Reply:
Even better, if you move this to the test file directly, it won't be part of libdali_operators.so

Reply:
I followed the pattern of already existing `Passthrough` operators, that is made specifically for testing:
![obraz](https://user-images.githubusercontent.com/7101064/214285230-4024a0ad-ac69-41be-bf71-14aa111ab74f.png)

They all are located in the same position in the build tree and they are not the part of `libdali_operators.so`. They are compiled directly to the `dali_operator_test.bin` binary, as it can be seen here:
```cmake
  add_executable(dali_operator_test
    ${DALI_OPERATOR_TEST_SRCS}
    ${DALI_ROOT}/dali/test/dali_test_config.cc
    ${DALI_ROOT}/dali/test/dali_operator_test_utils.cc
    ${DALI_ROOT}/dali/test/operators/passthrough_with_trace.cc
    ${DALI_ROOT}/dali/test/operators/passthrough_input.cc)
```

Is that OK? I believe having namespace might be nice, however other `Passthrough...` operators do not have the namespace there, so maybe it would be better to retain consistency between them?

Reply:
Sure, I didn't see the `test` part in the path